.github/workflows/tox-tests.yml

```
@@ -68,7 +68,8 @@ jobs:
       env:
         TOXENV: ansible${{ matrix.ansible-version }}
     - name: Archive logs
-      uses: actions/upload-artifact@v1
+      uses: actions/upload-artifact@v2
+      if: ${{ always() }}
```

The idiomatic way to having if-clauses is as follows:
```suggestion
      if: always()
```